subgraphs/bancor-v3/README.md

```diff
@@ -0,0 +1,313 @@
+
+_TVL USD = SUM of TVL USD IN ALL LIQUIDITY POOLS IN PROTOCOL_
+
+Example:
+
+Let’s assume that there are 4 liquidity pools in the Bancor V3 protocol which are: ETH, BNT, DAI and LINK.
+
+The pools have the following TVL USD each:
+
+* ETH: $1,000,000 USD
+* BNT: $1,800,000 USD
+* DAI: $2,000,000 USD
+* LINK: $1,500,000 USD
+
+In this case, the Total Value Locked USD for the protocol would be: 
+
+_TVL USD = ETH TVL USD + BNT TVL USD + DAI TVL USD + LINK TVL USD = $6,300,000_
+
+**Protocol Controlled Value USD**
+
+Protocol Controlled Value is relevant for Bancor V3. 
+
+The formula for the protocol controlled value for Bancor V3 will be as follows:
+
+_Total Protocol Controlled Value USD = Amount of vBNT Burned * Latest price of BNT in USD_
+
+This is how the official Bancor dune analytics dashboard estimated the BNT Value Locked Forever. 
+
+Reference links
+
+* Dune Analytics Dashboard: [https://dune.com/Bancor/bancor_1](https://dune.com/Bancor/bancor_1)
+* Dune Analytics BNT Value Locked Forever Query: [https://dune.com/queries/30032/73935](https://dune.com/queries/30032/73935)
+* Bancor Vortex Burner: [Bancor Vortex Burner](https://docs.bancor.network/guides/triggering-the-bancor-vortex-burner)
+* Bancor V3 Proposal Document - see Bancor Vortex section: [Bancor V3 Proposal Document](https://docs.google.com/document/d/11UeMYaI_1CWdf_Nu6veUO-vNB5uX-FcVRqTSU-ziDRk/edit#)
+
+**Total Revenue USD**
+
+_Total Revenue USD = Supply Side Revenue USD + Protocol Revenue USD_
+_Total Revenue USD = Total Trading Fees USD + Total Flash Loan Fees USD (LP side) + Total Exit Fees USD + Total Flash Loan Fees (Protocol Side) + Total Vortex Fees USD_
+
+**Supply Side Revenue USD**
+
+_Total Supply Side Revenue USD = Total Trading Fees USD + Total Flash Loan Fees USD_
+
+_Total Supply Side Revenue USD = Total BNT to TKN Trading Fees USD + Total TKN to BNT Trading Fees USD + Total TKN to TKN Trading Fees USD + Total Flash Loan Fees USD_
+
+**Protocol Revenue USD**
+
+_Protocol Revenue USD = Total Exit Fees from Liquidity Pools USD + Total Flash Loan Fees USD + Total Vortex Fees USD_
+
+
+## Pool-Level Metrics
+
+**Pool Total Value Locked USD**
+
+Pool Total Value Locked USD is calculated summing the total USD value of the tokens in a given liquidity pool
+
+_Pool Total Value Locked USD = Number of tokens in liquidity pool * price of token in USD_
+
+Example:
+
+Let’s assume that there are 1,500,000 LINK in the LINK liquidity pool on Bancor V3 and the current price of LINK at the time of calculation is $8.
+
+_Pool Total Value Locked USD (LINK Liquidity Pool) = Number of tokens in liquidity pool * price of token USD = 1,500,000 * $8 = $12,000,000 USD_
+
+**Reward Tokens & Reward Token Emissions Amount**
+
+**BNT Pool Reward Token Emissions**
+
+In Bancor V3 there will be a maximum of 40,000,000 BNT that will be distributed over an infinite time period. 
```

Comment:
Breakdown of total BNT rewards:
30M BNT will be distributed over an infinite time period to BNT LPs
10M BNT to be distributed to TKN LPs, not necessarily under an exponential decay emission model i.e. over an infinite time period, as we also have the flat/linear emissions model mentioned below

Reply:
Appreciate this info, thank you.